io.beekeeperstudio.Studio.metainfo.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.beekeeperstudio.Studio</id>
+
+    <name>Beekeeper Studio</name>
+    <summary>The SQL Editor and Database Manager Of Your Dreams</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <developer_name>Beekeeper Studio, Inc</developer_name>
+
+    <description>
+        <p>
+            <i>NOTE: This wrapper is not verified by, affiliated with, or supported by Beekeeper Studio, Inc.</i>
```

Comment:
Why is this disclaimer needed if it's licensed under GPL3?

Reply:
It's convention for proprietary apps we have no redistribution rights for. It's not supposed to be used for free software.

Reply:
The trademark, if there is one, is covered by nominative fair use.

Reply:
It's what we've been through already with some vendors requesting removal, yes.